autoload/ale/fix/registry.vim

```diff
@@ -23,6 +23,11 @@ let s:default_registry = {
 \       'description': 'Apply prettier-standard to a file.',
 \       'aliases': ['prettier-standard'],
 \   },
+\   'elm-format': {
+\       'function': 'ale#fixers#elm_format#Fix',
+\       'suggested_filetypes': ['elm'],
+\       'description': 'Apply elm-format to a file.',
```

Comment:
What you should do here is remove the `format` key below, and add `'aliases': ['format'],` here to run the fixer with the old name.

Reply:
Ah, cool! I should have guessed that there was an existing mechanism for defining aliases. Fixed this now.